code/src/java/pcgen/gui2/converter/panel/WriteDirectoryPanel.java

```diff
@@ -133,9 +113,9 @@ public boolean returnAllowed()
 	public void setupDisplay(JPanel panel, final CDOMObject pc)
 	{
 		panel.setLayout(layout);
-		JLabel label = new JLabel(
+		Component label = new JLabel(
 				"Please select the Directory where Converted files should be written: ");
-		JButton button = new JButton("Browse...");
+		AbstractButton button = new JButton("Browse...");
```

Comment:
I think this is another point for discussion.  I get why we use List and Set rather than ArrayList, et al.  But I'm not sure I'm fond of always jumping down to Component and AbstractButton...